1-starter/package.json

```diff
@@ -8,9 +8,9 @@
   "scripts": {
     "build": "tsc",
     "build:changelog": "npx @discordx/changelog --root=src",
-    "dev": "node --loader ts-node/esm/transpile-only src/main.ts",
-    "serve": "node build/main.js",
-    "start": "nodemon --exec node --loader ts-node/esm/transpile-only src/main.ts"
+    "dev": "ts-node-esm src/main.ts",
```

Comment:
why?

Reply:
previous one was unnecessarily long when there was already a script provided by ts-node aka ts-node-esm

Reply:
looks like it was just renamed.

Reply:
that's good, I didn't knew